packages/core/src/lib/disk.ts

```diff
@@ -147,3 +151,15 @@ export class DiskWrapper {
     })
   }
 }
+
+const PUBLIC_PATH = 'uploads'
+
+export const disk = new DiskWrapper({
+  local: {
+    publicPath: PUBLIC_PATH,
+    location: path.join(
+      path.dirname(fileURLToPath(import.meta.url)),
+      `../../public/${PUBLIC_PATH}`,
```

Comment:
This path is ok now? Before was `apps/web/public/uploads`

Reply:
hmm i'm not sure this is a valid approach anyways since this file gets bundled in the production bundle in a completely different location so...

Reply:
Production doesn't matter if we us S3. But in develop the app using the disk has to have access to the uploads directory. Give a try, just asking maybe this way works

Reply:
it has to work in production OSS product anyone could be using local disk driver in production